src/redux-persist-cookie-storage.js

```diff
@@ -21,12 +22,27 @@ CookieStorage.prototype.getItem = function (key, callback) {
 }
 
 CookieStorage.prototype.setItem = function (key, value, callback) {
-  this.cookies.set(this.keyPrefix + key, value);
+  var expiration = null;
+
+  if (this.expiration !== null) {
+    var expires = this.expiration.default;
+
+    for (var n in this.expiration) {
+      if (key === n) {
+        expires = this.expiration[n];
+      }
+      expiration = {
+        expires
+      };
+    }
+  }
```

Comment:
`expires` should be `undefined` unless when the value is falsy, because Cookies-js expects `undefined` to indicate session duration.

It is not necessary to iterate through the expiration object. We know what property we're looking for so you can use the `[]` operator.